locales/ru.ts

```diff
@@ -281,6 +281,7 @@ export const ru = {
   },
 
   logOut: {
+    logOut: 'Выход',
```

Comment:
Also missing is the translation of the modal window buttons into `en` and `ru` locales.